DEVELOPING.md

```diff
@@ -543,7 +543,7 @@ following:
 
 ```go
 // 1. Create the Test.
-pingTester := ping.NewPing(defaultTimeout, targetPodIPAddress, count)
+pingTester := ping.NewPing(targetPodIPAddress, count)
```

Comment:
The whole file needs an update or be deleted for now. The whole code changed

Reply:
Yes, I agree.  Do we want to do that in this PR or a subsequent change?

Reply:
I would rather just delete this file now and create a new one in a future PR as needed. Current content is not applicable to the new project

Reply:
Sounds good, I'll just remove this file instead.